cves/kernel/CVE-2021-29647.yml

```diff
@@ -358,8 +351,8 @@ forgotten_check:
     Answer must be true or false.
     Write a note about how you came to the conclusions you did, regardless of
     what your answer was.
-  answer:
-  note:
+  answer: false
+  note: This was a forgoten step in initialization.
```

Comment:
Possibly elaborate on what specific stem was overlooked in the initialization process